gui/public/i18n/en/translation.ftl

```diff
@@ -814,6 +814,8 @@ onboarding-automatic_proportions-check_height-title = Check your height
 onboarding-automatic_proportions-check_height-description = We use your height as a basis of our measurements by using the headset's (HMD) height as an approximation of your actual height, but it's better to check if they are right yourself!
 # All the text is in bold!
 onboarding-automatic_proportions-check_height-calculation_warning = Please press the button while standing <u>upright</u> to calculate your height. You have 3 seconds after you press the button!
+onboarding-automatic_proportions-check_height-guardian_tip = If you are using an standalone VR, be sure to have your guardian/
+    boundary turned on so your height is correct!
```

Comment:
```suggestion
onboarding-automatic_proportions-check_height-guardian_tip = If you are using a standalone VR headset, make sure to have your guardian/
    boundary turned on so that your height is correct!
```